networkx/algorithms/operators/all.py

```diff
@@ -50,6 +50,19 @@ def union_all(graphs, rename=()):
     If a graph attribute is present in multiple graphs, then the value
     from the last graph in the list with that attribute is used.
 
+    Examples
+    --------
+    >> # Initializing two graphs
+    >> G1 = nx.Graph()
+    >> G2 = nx.Graph()
+    >> # Adding edges
+    >> G1.add_edges_from([(1, 2), (2, 3)])
+    >> G2.add_edges_from([(4, 5), (5, 6)])
+    >> # Finding union of G1 and G2
+    >> result_graph = union_all([G1, G2])
+    >> print("Nodes:", result_graph.nodes())
+    Nodes: [1, 2, 3, 4, 5, 6]
```

Comment:
IMO it's best to have basic usage examples be as minimal as possible, so I'd advocate for condensing graph creation, removing the comments, and eliminating printing where possible. There's also a slight syntax issue: the Python prompt indicator should be `>>>`:

```suggestion
    >>> G1 = nx.Graph([(1, 2), (2, 3)])
    >>> G2 = nx.Graph([(4, 5), (5, 6)])
    >>> U = nx.union_all([G1, G2])
    >>> U.edges()
    EdgeView([(1, 2), (2, 3), (4, 5), (5, 6)])
```